integration-tests/tests/src/tests/sync/user.ts

```diff
@@ -39,17 +39,17 @@ function expectIsUSer(user: Realm.User) {
   expect(user).instanceOf(Realm.User);
 }
 
-function expectIsSameUser(value: Realm.User, user: Realm.User | null) {
+function expectIsSameUser(value: Realm.User, user: Realm.User<any, any> | null) {
   expectIsUSer(value);
```

Comment:
I think there's a typo here?  `expectIsUSer` -> `expectIsUser`

Reply:
Yes, I'll fix this